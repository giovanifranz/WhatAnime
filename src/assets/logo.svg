<svg width="227" height="76" viewBox="0 0 227 76" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M37.9782 75.4777C58.7033 75.4777 75.5043 58.6767 75.5043 37.9516C75.5043 17.2265 58.7033 0.425537 37.9782 0.425537C17.2531 0.425537 0.452148 17.2265 0.452148 37.9516C0.452148 58.6767 17.2531 75.4777 37.9782 75.4777Z" fill="#F5DF4D"/>
<path d="M72.1961 53.3781C74.3205 48.6727 75.5033 43.4501 75.5033 37.9516C75.5042 17.2264 58.7033 0.425537 37.9781 0.425537C32.4796 0.425537 27.257 1.60831 22.5516 3.73272C46.2282 11.0183 64.9115 29.7006 72.1961 53.3781Z" fill="#DDC945"/>
<path d="M24.5511 28.4381H26.9973V21.2112H28.9365V31.4546H26.9973V30.5872H20.151V31.4546H18.2118V21.2112H20.151V28.4381H22.6128V18.213H24.5511V28.4381Z" fill="#939597"/>
<path d="M22.0704 37.4151H28.8966V39.5652H28.1823V41.7694C28.1823 42.4204 28.1282 42.9888 28.021 43.4766C27.9128 43.9644 27.7331 44.3715 27.4855 44.6961C27.237 45.0216 26.9125 45.2691 26.5127 45.4369C26.1129 45.6056 25.6261 45.69 25.053 45.69H24.777L24.3232 43.5399H25.02C25.5509 43.5399 25.9094 43.3959 26.0983 43.1071C26.2881 42.8174 26.3834 42.3718 26.3834 41.7703V39.5661H22.0695V41.5172C22.0695 42.2883 21.9852 43.0585 21.8174 43.8296C21.6496 44.5998 21.3993 45.3287 21.0637 46.0155L19.3931 45.1123C19.7076 44.5466 19.9313 43.9708 20.0661 43.3868C20.2018 42.8027 20.2697 42.1847 20.2697 41.5347V39.5652H18.2094V37.4151H20.2697V33.0077H22.0695V37.4151H22.0704Z" fill="#939597"/>
<path d="M18.2104 59.0746L23.0533 47.5669L24.7551 48.4343L20.8326 57.6471L25.6417 56.2378L24.2554 52.86L25.9919 52.011L29.1973 59.6706L27.4773 60.5196L26.4751 58.2073L18.9292 60.2849L18.2104 59.0746Z" fill="#939597"/>
<path d="M29.1973 67.4384H24.844V68.5588C24.844 69.1493 24.8009 69.8113 24.7101 70.5457C24.6212 71.2801 24.4158 71.9852 24.0949 72.6591C23.7722 73.3339 23.2926 73.9299 22.6572 74.4479C22.0209 74.966 21.1462 75.3098 20.0386 75.4776L19.5041 73.3101C20.2156 73.3101 20.7776 73.1506 21.1884 72.8315C21.5992 72.5124 21.9072 72.1117 22.1126 71.6295C22.318 71.1481 22.4436 70.6337 22.4922 70.0854C22.5408 69.538 22.5637 69.0292 22.5637 68.5597V67.4393H18.2104V65.2892H22.5637V62.0737H24.8431V65.2892H29.1964V67.4384H29.1973Z" fill="#939597"/>
<path d="M30.6981 21.4478L31.8222 18.213L32.2183 18.4569L31.307 21.0462L32.4237 20.6501L32.1019 19.7011L32.5053 19.4618L33.2498 21.6156L32.8501 21.854L32.6181 21.2039L30.865 21.7879L30.6981 21.4478Z" fill="#939597"/>
<path d="M31.2005 27.8511H30.698V24.576H31.6663V24.1295H32.1733V24.576H33.2497V25.1803H33.1415V26.749C33.1415 26.9315 33.1259 27.092 33.0966 27.2286C33.0663 27.3661 33.0168 27.4798 32.9472 27.5715C32.8775 27.6632 32.7867 27.7329 32.6758 27.7796C32.5639 27.8273 32.4282 27.8511 32.2687 27.8511H32.2008L32.0743 27.2469H32.2595C32.4071 27.2469 32.508 27.2066 32.5602 27.125C32.6134 27.0434 32.64 26.9187 32.64 26.749V25.1803H31.2014V27.8511H31.2005Z" fill="#939597"/>
<path d="M30.9658 30.5579H32.982V31.1621H32.2173V33.3406H33.2497V33.9448H30.698V33.3406H31.7259V31.1621H30.9658V30.5579Z" fill="#939597"/>
<path d="M31.1336 39.7797H30.698V36.7733H31.7066V36.3167H32.1265V36.7733H33.2497V37.3821H33.136V38.8802C33.136 38.9958 33.1278 39.1085 33.1104 39.2204C33.0939 39.3323 33.0636 39.434 33.0196 39.5248C32.9765 39.6165 32.9178 39.6935 32.8435 39.7559C32.7684 39.8191 32.6748 39.8604 32.563 39.8806L32.453 39.2763C32.5391 39.2598 32.6024 39.2186 32.6409 39.1544C32.6803 39.0902 32.6996 38.9985 32.6996 38.8802V37.3821H32.1265V40.2822H31.7066V37.3821H31.1336V39.7797Z" fill="#939597"/>
<path d="M30.7806 42.745H32.7555V43.3492H31.5517V43.9489H33.2497V44.5586H31.5517V45.0308C31.5517 45.1995 31.5755 45.3251 31.6213 45.4067C31.6681 45.4874 31.7589 45.5287 31.8937 45.5287H33.1012L32.9902 46.1329H31.8854C31.7396 46.1329 31.6158 46.1091 31.515 46.0614C31.4132 46.0137 31.3307 45.944 31.2674 45.8533C31.2042 45.7625 31.1583 45.6479 31.1308 45.5103C31.1033 45.3728 31.0896 45.2133 31.0896 45.0308V44.5586H30.698V43.9489H31.0896V43.3492H30.7806V42.745Z" fill="#939597"/>
<path d="M50.0351 18.0498C42.1188 18.0498 35.7006 24.467 35.7006 32.3843C35.7006 40.3015 42.1188 46.7188 50.0351 46.7188C57.9515 46.7188 64.3696 40.3015 64.3696 32.3843C64.3696 24.467 57.9515 18.0498 50.0351 18.0498ZM50.0351 41.2432C45.1427 41.2432 41.1762 37.2768 41.1762 32.3843C41.1762 27.4918 45.1427 23.5254 50.0351 23.5254C54.9276 23.5254 58.894 27.4918 58.894 32.3843C58.894 37.2768 54.9285 41.2432 50.0351 41.2432Z" fill="#939597"/>
<path d="M52.1441 37.8599C55.1682 37.8599 57.6197 35.4084 57.6197 32.3843C57.6197 29.3602 55.1682 26.9087 52.1441 26.9087C49.12 26.9087 46.6685 29.3602 46.6685 32.3843C46.6685 35.4084 49.12 37.8599 52.1441 37.8599Z" fill="#939597"/>
<path d="M86.2151 32.4888V8.41202L89.5133 6.96906L89.4927 32.4063C89.4927 33.0935 89.7195 33.6775 90.173 34.1585C90.6265 34.6395 91.1624 34.88 91.7808 34.88C92.4267 34.88 92.9764 34.6395 93.4299 34.1585C93.8972 33.6638 94.1308 33.0866 94.1308 32.427C94.1308 32.427 94.1308 32.4201 94.1308 32.4063L94.1514 8.41202L97.4496 6.96906L97.429 32.4063C97.429 33.0935 97.6557 33.6775 98.1092 34.1585C98.5627 34.6395 99.0987 34.88 99.7171 34.88C100.349 34.88 100.899 34.6395 101.366 34.1585C101.833 33.6775 102.067 33.0935 102.067 32.4063L102.088 8.41202L105.386 6.96906V32.5094C105.386 34.0486 104.836 35.3678 103.737 36.4672C102.637 37.5666 101.311 38.1163 99.7583 38.1163C98.1505 38.1163 96.8243 37.5804 95.7799 36.5085C94.6805 37.5804 93.3612 38.1163 91.8221 38.1163C90.173 38.1163 88.8331 37.5804 87.8024 36.5085C86.758 35.4228 86.2289 34.0829 86.2151 32.4888ZM110.209 38.1163V8.41202L113.508 6.96906V20.9039H125.505V8.41202L128.803 6.96906V36.6734L125.505 38.1163V24.2021H113.508V36.6734L110.209 38.1163ZM133.915 38.1163V16.1421C133.915 13.6135 134.822 11.456 136.636 9.66945C138.45 7.86919 140.642 6.96906 143.212 6.96906C145.782 6.96906 147.974 7.86919 149.788 9.66945C151.602 11.456 152.509 13.6135 152.509 16.1421V36.6734L149.211 38.1163V24.2021H137.213V36.6734L133.915 38.1163ZM137.213 20.9039H149.211V16.0803C149.197 14.4312 148.606 13.0295 147.438 11.8751C146.256 10.707 144.847 10.123 143.212 10.123C141.563 10.123 140.147 10.707 138.966 11.8751C137.797 13.0295 137.213 14.4106 137.213 16.0185V16.0803V20.9039ZM165.269 38.1163V10.6795H155.992L157.435 7.38133H177.843L176.4 10.6795H168.567V36.6734L165.269 38.1163ZM86.5037 71.1163V49.1421C86.5037 46.6135 87.4107 44.456 89.2247 42.6695C91.0387 40.8692 93.2307 39.9691 95.8005 39.9691C98.3703 39.9691 100.562 40.8692 102.376 42.6695C104.19 44.456 105.097 46.6135 105.097 49.1421V69.6734L101.799 71.1163V57.2021H89.8019V69.6734L86.5037 71.1163ZM89.8019 53.9039H101.799V49.0803C101.785 47.4312 101.194 46.0295 100.026 44.8751C98.8445 43.707 97.4359 43.123 95.8005 43.123C94.1514 43.123 92.7359 43.707 91.5541 44.8751C90.386 46.0295 89.8019 47.4106 89.8019 49.0185V49.0803V53.9039ZM110.209 71.1163V49.1421C110.209 46.6135 111.116 44.456 112.93 42.6695C114.744 40.8692 116.936 39.9691 119.506 39.9691C122.076 39.9691 124.268 40.8692 126.082 42.6695C127.896 44.456 128.803 46.6135 128.803 49.1421V69.6734L125.505 71.1163V49.0803C125.491 47.4312 124.9 46.0295 123.732 44.8751C122.55 43.707 121.142 43.123 119.506 43.123C117.857 43.123 116.442 43.707 115.26 44.8751C114.092 46.0295 113.508 47.4106 113.508 49.0185C113.508 49.046 113.508 49.0666 113.508 49.0803V69.6734L110.209 71.1163ZM135.667 40.3813H152.2L150.757 43.6795H144.861V67.4059H152.2L150.757 70.7041H134.224L135.667 67.4059H141.563V43.6795H134.224L135.667 40.3813ZM157.332 71.1163V45.576C157.332 44.0368 157.882 42.7176 158.981 41.6182C160.081 40.5188 161.407 39.9691 162.96 39.9691C164.568 39.9691 165.894 40.505 166.938 41.5769C168.038 40.505 169.357 39.9691 170.896 39.9691C172.545 39.9691 173.885 40.505 174.916 41.5769C175.96 42.6626 176.489 44.0025 176.503 45.5966V69.6734L173.205 71.1163L173.225 45.6791C173.225 44.9919 172.999 44.4079 172.545 43.9269C172.092 43.4459 171.556 43.2054 170.937 43.2054C170.291 43.2054 169.742 43.4459 169.288 43.9269C168.821 44.4216 168.587 44.9988 168.587 45.6584V45.6791L168.567 69.6734L165.269 71.1163L165.289 45.6791C165.289 44.9919 165.062 44.4079 164.609 43.9269C164.155 43.4459 163.62 43.2054 163.001 43.2054C162.369 43.2054 161.819 43.4459 161.352 43.9269C160.885 44.4079 160.651 44.9919 160.651 45.6791L160.631 69.6734L157.332 71.1163ZM186.274 57.2021V67.4059H200.951L199.508 70.7041H182.976V41.8037L186.274 40.3607L198.271 40.3813L196.828 43.6795H186.274V53.9039H198.271L196.828 57.2021H186.274ZM220.348 49.1009C220.348 47.4793 219.764 46.0707 218.596 44.8751C217.428 43.707 216.013 43.123 214.35 43.123C212.728 43.123 211.32 43.707 210.124 44.8751C208.942 46.0295 208.351 47.3762 208.351 48.9154C208.351 48.9704 208.351 49.0253 208.351 49.0803V49.2864L205.053 50.7294V49.1421C205.053 46.6548 205.96 44.4972 207.774 42.6695C209.561 40.8692 211.752 39.9691 214.35 39.9691C216.933 39.9691 219.125 40.8623 220.926 42.6488C222.712 44.4216 223.605 46.6135 223.605 49.2246V49.2658C223.605 51.8082 222.588 53.9451 220.555 55.6767C220.39 55.9103 218.727 56.9135 215.566 58.6863C214.769 59.1123 214.295 59.6689 214.144 60.356C214.089 60.5209 214.089 60.8919 214.144 61.4691L210.845 62.9121C210.845 62.8021 210.866 61.8264 210.907 59.9849C210.935 58.4046 212.46 56.7967 215.484 55.1613C216.926 54.378 217.971 53.7253 218.617 53.203C219.771 52.2686 220.348 50.9012 220.348 49.1009ZM210.66 70.333C210.138 69.7971 209.877 69.1512 209.877 68.3953C209.877 67.6395 210.138 67.0005 210.66 66.4783C211.182 65.9423 211.814 65.6743 212.556 65.6743C213.298 65.6743 213.931 65.9423 214.453 66.4783C214.975 67.0005 215.236 67.6395 215.236 68.3953C215.236 69.1512 214.975 69.7971 214.453 70.333C213.931 70.8552 213.298 71.1163 212.556 71.1163C211.814 71.1163 211.182 70.8552 210.66 70.333Z" fill="#F5DF4D"/>
</svg>
