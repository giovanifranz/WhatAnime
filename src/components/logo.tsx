import Image from 'next/image'
import Link from 'next/link'

import WhatAnime from '@/assets/logo.svg'

export function Logo() {
  return (
    <Link href="/">
      <Image src={WhatAnime} alt="WhatAnime" width={227} height={76} />
    </Link>
  )
}
