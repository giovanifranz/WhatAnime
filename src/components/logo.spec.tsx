import { render } from '@/test/test-utils'

import { Logo } from './logo'

describe('Logo', () => {
  it('Deve renderizar o Logo corretamente', () => {
    const { asFragment } = render(<Logo />)
    expect(asFragment()).toMatchSnapshot()
  })
})
