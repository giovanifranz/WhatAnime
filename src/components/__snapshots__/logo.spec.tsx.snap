// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Logo > Deve renderizar o Logo corretamente 1`] = `
<DocumentFragment>
  <script>
    !function(){try{var d=document.documentElement,c=d.classList;c.remove('light','dark');var e=localStorage.getItem('theme');if('system'===e||(!e&&false)){var t='(prefers-color-scheme: dark)',m=window.matchMedia(t);if(m.media!==t||m.matches){d.style.colorScheme = 'dark';c.add('dark')}else{d.style.colorScheme = 'light';c.add('light')}}else if(e){c.add(e|| '')}else{c.add('dark')}if(e==='light'||e==='dark'||!e)d.style.colorScheme=e||'dark'}catch(e){}}()
  </script>
  <a
    href="/"
  >
    <img
      alt="WhatAnime"
      data-nimg="1"
      decoding="async"
      height="76"
      loading="lazy"
      src="/src/assets/logo.svg"
      style="color: transparent;"
      width="227"
    />
  </a>
</DocumentFragment>
`;
